packages/frontend-2/lib/projects/composables/previewImage.ts

```diff
@@ -5,13 +5,39 @@ import { useSubscription } from '@vue/apollo-composable'
 import { useLock } from '~~/lib/common/composables/singleton'
 import PreviewPlaceholder from '~~/assets/images/preview_placeholder.png'
 import { isValidBase64Image } from '@speckle/shared/images/base64'
+import { nanoid } from 'nanoid'
+
+/**
+ * Eager loading previews ensures a better LCP score, but also hits the preview endpoint more often.
```

Comment:
@andrewwallacespeckle loading the actual final preview ASAP really does end up being the only solution for the LCP issue. so there's a few extra safeguards in place to eager load only on initial page load (not for subsequently loaded previews) and only for up to 20 previews